feature-group-definitions/background-gradients.yml

```
@@ -1,4 +1,5 @@
 name: Background gradients
+description: The CSS gradient functions create background images consisting of progressive transitions between two or more colors.
```

```suggestion
description: The `linear-gradient()`, `radial-gradient()`, and `conic-gradient()` CSS functions create backgrounds consisting of progressive transitions between two or more colors.
```

---

Since "transitions" are something else, how about this:

```suggestion
description: The `linear-gradient()`, `radial-gradient()`, and `conic-gradient()` CSS functions create backgrounds that progress between two or more colors.
```